share/spice/hearthstone/hearthstone.js

```diff
@@ -0,0 +1,53 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_hearthstone = function(api_result){
+
+        // Validate the response
+        if (!api_result || api_result.error || !api_result.Has_name || !api_result.page) {
+            return Spice.failed('hearthstone');
+        }
+        
+        // Card Data
+        // May (but very unlikely) return <undefined> values for : image_url, Has_card_type, Has_rarity,
+        // Has_class, Has_mana_cost
+        var card = {
```

Comment:
This should be handled with a `normalize` function: https://duck.co/duckduckhack/spice_displaying#data-normalization